### ISSUE TITLE
Migrate deprecated symfony AbstractCache to AbstractAdapter

### ISSUE THREAD

**Issue by emteknetnz** [silverstripe/silverstripe-framework#9958]
A couple of community members noted that colymba/gridfield-bulk-editing-tools 3.0.0 was throwing (unrelated) symfony deprecation warnings with framework 4.7.3 that was breaking the upload field field.

I replicated this locally with `SS_ENVIRONMENT_TYPE="live"`

```
Deprecated: The "Symfony\Component\Cache\Simple\AbstractCache" class is deprecated since Symfony 4.3, use "Symfony\Component\Cache\Adapter\AbstractAdapter" and type-hint for "Symfony\Contracts\Cache\CacheInterface" instead. in /var/www/vendor/symfony/cache/Simple/AbstractCache.php on line 23
```
- https://github.com/silverstripe/silverstripe-framework/issues/9955
- https://github.com/colymba/GridFieldBulkEditingTools/issues/214

The reason there was that colymba/gridfield-bulk-editing-tools was making so much of a fuss about deprecated classes that it caused itself to break

However we still need to migrate the symfony AbstractCache to AbstractAdapter for the rest of framework

**Comment by dhensby** [comment]
Can this be closed now?

**Comment by emteknetnz** [comment]
Nope.  We still need to migrate off the deprecated Symfony class at some point.
